very_good_docs_site/__brick__/{{project_name.snakeCase()}}/docusaurus.config.js

```diff
@@ -1,8 +1,8 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
 
-const lightCodeTheme = require('prism-react-renderer/themes/vsLight');
-const darkCodeTheme = require('prism-react-renderer/themes/vsDark');
+const lightCodeTheme = require('prism-react-renderer').themes.vsLight;
+const darkCodeTheme = require('prism-react-renderer').themes.vsDark;
```

Comment:
Why did this change?

Reply:
There was a change in their API. It fails if this isn't changed to access the theme this way now. We have the same change in the other doc sites, like Dart Frog, as well.